bootstrap/pkg/kfapp/ksonnet/ksonnet.go

```diff
@@ -437,10 +437,16 @@ func (ksApp *ksApp) envSet(envName string, host string) error {
 		actions.OptionAppRoot: ksApp.ksRoot(),
 		actions.OptionEnvName: ksApp.KsEnvName,
 		actions.OptionServer:  host,
+		actions.OptionOverride: true,
 	})
 	if err != nil {
 		return fmt.Errorf("There was a problem setting ksonnet env: %v", err)
 	}
+	loadApp, loadErr := app.Load(afero.NewOsFs(), ksApp.KApp.HTTPClient(), ksApp.ksRoot())
```

Comment:
I see. So I'm guessing RunEnvSet is just modifying the config files on disks and we need to explicitly load the app to pick up the changes.